src/style.css

```diff
@@ -1,3 +1,151 @@
+html {
+  .roboto-thin {
+    font-family: Roboto, sans-serif;
+    font-weight: 100;
+    font-style: normal;
+  }
+
+  .roboto-light {
+    font-family: Roboto, sans-serif;
+    font-weight: 300;
+    font-style: normal;
+  }
+
+  .roboto-regular {
+    font-family: Roboto, sans-serif;
+    font-weight: 400;
+    font-style: normal;
+  }
+
+  .roboto-medium {
+    font-family: Roboto, sans-serif;
+    font-weight: 500;
+    font-style: normal;
+  }
+
+  .roboto-bold {
+    font-family: Roboto, sans-serif;
+    font-weight: 700;
+    font-style: normal;
+  }
+
+  .roboto-black {
+    font-family: Roboto, sans-serif;
+    font-weight: 900;
+    font-style: normal;
```

Comment:
Avoid using tag names for styling (except `html` and `body`). Instead of applying styles directly to the `html` element, create a separate class for the font styles and apply it to the elements that require it. This will also help to avoid repeating `font-family: Roboto, sans-serif;` for each class.